Block/SocialLinks.php

```diff
@@ -34,7 +34,9 @@ class SocialLinks extends Template implements BlockInterface {
         "pinterest" => "//www.pinterest.com/",
         "instagram" => "//www.instagram.com/",
         "youtube"   => "//www.youtube.com/",
-        "snapchat"  => "//www.snapchat.com/add/"
+        "snapchat"  => "//www.snapchat.com/add/",
+        "tiktok"    => "//www.tiktok.com/@",
```

Comment:
This is my only point of contention, however it follows the convention for all other options where it tells them not to add #, @ or / characters.